lib/fabrik/DirectorInstance.js

```diff
@@ -73,9 +76,12 @@ class DirectorInstance extends BaseInstance {
       .try(() => {
         switch (operation.type) {
         case 'create':
-          return this
-            .createSecurityGroup(operation.space_guid)
-            .tap(() => operation.state === CONST.OPERATION.SUCCEEDED ? this.scheduleAutoUpdate() : {});
+          if (operation.context.platform === 'cloudfoundry') {
+            return this
+              .createSecurityGroup(operation.context.space_guid)
+              .tap(() => operation.state === CONST.OPERATION.SUCCEEDED ? this.scheduleAutoUpdate() : {});
```

Comment:
K8s instances dont need to be auto-updated?

Reply:
As stated above, kept it only for temporary purpose. With SF2.0, only scheduleautoupdate will be done here.